src/BuildScriptGeneratorCli/CliEnvironmentSettings.cs

```diff
@@ -17,6 +17,8 @@ public class CliEnvironmentSettings
         private const string DisableNodeJsEnvVarName = "DISABLE_NODEJS_BUILD";
         private const string EnableMultiPlatformBuildEnvVarName = "ENABLE_MULTIPLATFORM_BUILD";
         private const string GitHubActionsEnvVarName = "GITHUB_ACTIONS";
+        private const string GitHubActionsBuildStartTimeEnvVarName = "GITHUB_ACTIONS_BUILD_START_TIME";
+        private const string GitHubActionsBuildEndTimeEnvVarName = "GITHUB_ACTIONS_BUILD_END_TIME";
```

Comment:
Would the following be appropriate?
`GITHUB_ACTIONS_BUILD_START_TIME` => `GITHUB_ACTIONS_BUILD_IMAGE_PULL_START_TIME`
`GITHUB_ACTIONS_BUILD_END_TIME` => `GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME`

Reply:
Thanks! I'll go for these names, they are specific and no ambiguity.